### ISSUE TITLE
[FEATURE REQUEST] Interactive Legends for Plotly Express Sunburst Charts

### ISSUE THREAD

**Issue by mlouis9** [plotly/plotly.py#4692]
## Motivation
I'm a huge fan of Plotly Express Sunburst charts, but when viewing data with lots of small proportions (that are meaningful), it can be difficult to expand individual wedges because they're so small.

## A Solution
I think it might be nice if there was a scrollable, clickable legend listing all of the inner wedges of the pie chart. I've included an [an example .html spec](https://drive.google.com/file/d/1uR65mgdVl3b7knshnk2dTLIXH1VQrDpO/view?usp=sharing) that I hacked together by wrapping the plotly html. Clicking on the label in the legend expands the corresponding wedge in the sunburst chart. I'm not sure how feasible something like this is, but I think it might be a nice feature.

**Comment by archmoj** [comment]
Is this the duplicate of https://github.com/plotly/plotly.js/issues/4760?

**Comment by mlouis9** [comment]
Not quite; the suggested addition is to deal with wedges that are too small to click on by hovering over them by adding an interactive legend showing the inner wedges of the sunburst. As far as I can tell by looking at the example in the linked issue, it doesn't implement the type of legend included in the html spec I linked [here](https://github.com/plotly/plotly.py/issues/4692#issue-2427490501) and still has issues with small wedges.
